Patch to allow building the Objective-C compiler for the AVR.
===================================================================
--- gcc/config/avr/avr.h.old	2007-08-23 15:18:31.015625000 -0600
+++ gcc/config/avr/avr.h	2007-08-23 15:19:17.687500000 -0600
@@ -53,7 +53,7 @@ extern int avr_mega_p;
 extern int avr_have_mul_p;
 extern int avr_asm_only_p;
 extern int avr_have_movw_lpmx_p;
-#ifndef IN_LIBGCC2
+#if !defined(IN_LIBGCC2) && !defined(IN_TARGET_LIBS)
 extern GTY(()) section *progmem_section;
 #endif
 
